README.md

```diff
@@ -14,13 +22,50 @@ npm install
 npm run dev
 ```
 
-### Type-Check, Compile and Minify for Production
+### Build
 
+Build the Docker-Container
+```sh
+docker build -t crosswordpuzzle-dev
+```
+And run it at port 8000 with
 ```sh
-npm run build
+docker run -d -p 8000:80 --name crosswordpuzzle-dev crosswordpuzzle-dev
 ```
 
-## Customize configuration
+To monitor, stop and remove the container you can use the following commands:
+```sh
+docker ps -a -f name=crosswordpuzzle-dev
+```
+```sh
+docker stop crosswordpuzzle-dev
+```
+```sh
+docker rm crosswordpuzzle-dev
+```
+
+## User manual
+
+Run the docker container with the following commang at port 8000:
+```sh
+docker run -d -p 8000:80 --name crosswordpuzzle ghcr.io/gamify-it/crosswordpuzzle:latest
+```
+Now you can access it at [http://localhost:8000](http://localhost:8000).  
+To acces it external replace localhost with your IP.  
+
+To monitor the container you can use the following commands:
+```sh
+docker ps -a -f name=crosswordpuzzle
+```
+To stop the container you can use the following commands:
+```sh
+docker stop crosswordpuzzle
+```
+To remove the container you can use the following commands:
+```sh
+docker rm crosswordpuzzle
+```
 
-See [Vite Configuration Reference](https://vitejs.dev/config/).
+### Screenshot
 
+![example auto genderated crosswordpuzzle](https://cdn.discordapp.com/attachments/946034796243595284/975526829773426758/unknown.png "crosswordpuzzle")
```

Comment:
I would either recommend
a) copy-pasting the image into the GitHub editor so that it will be stored on GitHubs servers.
or
b) Adding a docs (or screenshots) directory where you upload the picture to be able to update it automatically.

Reply:
Yeah I will do a). I didn't know the possibility to host the images such easy at github